docs/en/development/build_cross_arm.md

```diff
@@ -0,0 +1,35 @@
+# How to Build ClickHouse on Linux for AARCH64 (ARM64) architecture
+
+This is for the case when you have Linux machine and want to use it to build `clickhouse` binary that will run on another Linux machine with AARCH64 CPU architecture. This is intended for continuous integration checks that run on Linux servers.
+
```

Comment:
I didn't understand relation between the ClickHouse building and CI that run on some unknown Linux servers.

Reply:
The point is that the purpose of this instruction is for some CI Build to work on Linux server (with x86-64 architecture). If you can rephrase it in a better way, you're welcome.